app/views/glossary/_tooltip.html.erb

```diff
@@ -1,5 +1,3 @@
-<span class='glossary-definition' >
-  <strong>Unfamiliar term?</strong>
-  <br />
-  <span>Click to read about it in the Security Self Defense glossary.</span>
+<span class='glossary-definition' id="<%= id %>" >
+  <%= truncate(term.body, length: 200, separator: ' ') { link_to "Read more", glossary_path(term.slug) } %>
```

Comment:
We should `strip_tags` here too, before truncating:

![glossary_term_tag](https://user-images.githubusercontent.com/6469642/37307699-fd06498c-25f8-11e8-9373-68995618d799.png)